lib/pacemaker/pcmk_cluster_queries.c

```diff
@@ -208,6 +208,17 @@ designated_controller_event_cb(pcmk_ipc_api_t *controld_api,
     event_done(data, controld_api);
 }
 
+/*!
+ * \internal
+ * \brief Process a \p pacemakerd status IPC event
```

Comment:
A completely unimportant stylistic note: I was torn on whether to use "`pacemakerd`" or "launcher" in doxygen blocks. We usually say "controller" and "scheduler" instead of "`pacemaker-controld` and "`pacemaker-schedulerd`". "Launcher" is the analog for "`pacemakerd`", and we use it in log messages (`pcmk_ipc_name()`).

However, it's used almost nowhere in the code or documentation right now so its meaning is probably less clear than "`pacemakerd`".